models/public/colorization-v2-pytorch/accuracy-check.yml

```diff
@@ -0,0 +1,22 @@
+evaluations:
+  - name: colorization-v2-pytorch
+    module: custom_evaluators.colorization_evaluator.ColorizationEvaluator
+    module_config:
+      network_info:
+        verification_network:
```

Comment:
it is unclear from config that it is required 2 models, please add section for colorization model too (it can have empty dictionary as parameters)

Reply:
Added